meta/implementation.md

```diff
@@ -73,4 +73,29 @@ you may define something like the following for the parameter:
 
 	]
 }
-```
\ No newline at end of file
+```
+
+## Date and Time manipulation
+
+Working with dates is a lot more complex than it seems to be at first sight. Issues arise especially with daylight saving times (DST), time zones, leap years and leap seconds.
+
+The date/time functions in openEO don't have any effect right now as only timestamps in UTC (with potential numerical time zone modifier) are supported.
+
+Leap years are implemented in a way that computations handle them gracefully. For example:
```

Comment:
You talk about leap years here, but all the examples are about the "month overflow" issue, which happens regardless of leap years.

An example that would involve leap years is `2020-02-29 + 1 year => 2020-02-28`, but even this can be explained with the "month overflow" behavior we are documenting here. The leap year problem is just a small side note I think.

Reply:
```suggestion
Month overflows, including the specific case of leap years, are implemented in a way that computations handle them gracefully. For example:
```